src/support/lockedpool.cpp

```diff
@@ -133,6 +133,12 @@ Arena::Stats Arena::stats() const
     return r;
 }
 
+bool Arena::addressInArena(void *ptr_in) const
+{
+    uintptr_t ptr = reinterpret_cast<uintptr_t>(ptr_in);
```

Comment:
I'm slightly allergic to pointer math, that's why the preference to cast to uintptr_t where possible. But sure, for comparisons it can't hurt.